network/network.py

```diff
@@ -48,7 +48,70 @@ def _generate_nodes(self):
     def refresh_network(self):
         self.nodes = self._generate_nodes()
 
+    def get_node_by_mac(self, mac_addr):
```

Comment:
i dont need this function - I need start_mitm(mac_addr) and stop_mitm(mac_addr) please :)

Reply:
This isnt for you, asshole